shared/feed-parser.js

```diff
@@ -110,6 +110,11 @@ const FeedParser = {
       if (!url) {
         url = getTextFromElement("guid:not([isPermaLink='false'])", item);
       }
+      if (!url) {
+        if (media && media.length) {
+          url = media[0].url;
+        }
+      }
```

Comment:
Wdyt of:

```suggestion
      if (!url && media && media.length) {
        url = media[0].url;
      }
```

Both works either way :)